modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types are consistent across all ClusterVersion and ClusterOperator conditions. Individual operators may also have additional condition types with operator-specific language.
+
+The condition type, as well as additional information about your operator, can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o yaml
+----
+
+
+* Available: 
+An operator with the condition type `Available` is functional and available in the cluster. If the status is `False`, at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+An operator with the condition type `Progressing` is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators do not report as `Progressing` when they are reconciling a previous known state. If the observed cluster state has changed and the operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
++
+* Degraded:
+An operator with the condition type `Degraded` has a current state does not match the desired state over a period of time. The period of time may vary by component, but a `Degraded` state represents persistent observation of an operator's condition.  As a result, an operator will not fluctuate in and out of the `Degraded` state.  
++
+There may be a different condition type if the transition from one state to another does not persist over a long enough period to report `Degraded`.  
+An operator will not report `Degraded` during the course of a normal upgrade.  An operator may report `Degraded` in response to a persistent infrastructure failure that requires eventual administrator intervention.
++
+[NOTE]
+====
+This condition type can seem alarming, but it is only an indication that that something may need investigation and adjustment. As long as the operator is available, the `Degraded` condition does not cause user workload failure or application downtime.
```

Comment:
This sounds maybe a little too conversational and/or wordy at the beginning for me. Maybe consider rewriting the first sentence to `This condition type is only an indication that that something may need investigation and adjustment.`? Not the biggest deal either way

Reply:
good point :)